### PR TITLE
Added basic structure

### PR DESCRIPTION
**Pull request by sebalp100**
# Added basic structure

For this initial milestone we added the following:

- Basic navbar
- Home page logo
- Different section links with react-router
- React - Redux linters

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <!--
```
Comment (SimonGideon) [inline public/index.html]: When creating a react-up, I suggest you get rid of the unnecessary lines of code before you start working on your project.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```
Comment (SimonGideon) [inline src/index.js]: It's a best practice to avoid unnecessary comments or commented codes to be pushed on github.

**Comment by sebalp100** [comment]
Changes required implemented 👍

File: src/index.js (hunk 2 above)
Comment (sebalp100) [reply src/index.js]: Thank you, I will delete the unnecessary comments.

File: public/index.html (hunk 1 above)
Comment (sebalp100) [reply public/index.html]: Thank you, I will delete the unnecessary comments.

**Review by SimonGideon — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @sebalp100  

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

giphy




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Optional Suggestion

- [ ] You have really given your best at this level of the project, I really like your efforts also I suggest you avoid React app unnecessary files and lines of code check the Inline comments for more information.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @SimonGideon .

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**